template_scripts/09_cleanup.sh

```diff
@@ -10,7 +10,7 @@ echo "--> Rebuilding rpm database..."
 chroot `pwd`/mnt /bin/rpm --rebuilddb 2> /dev/null
 
 if [ -x mnt/usr/bin/dnf ]; then
-    chroot mnt dnf clean all
+    chroot `pwd`/mnt dnf clean all
```

Comment:
Prefer `$(...)` syntax instead of the deprecated backticks

Reply:
Agree, fixed